apps/evm/src/clients/api/queries/useGetPools/__tests__/__snapshots__/index.spec.ts.snap

```diff
@@ -76,8 +76,8 @@ exports[`useGetPools > returns pools in the correct format 1`] = `
           "borrowPointDistributions": [],
           "borrowTokenDistributions": [
             {
-              "apyPercentage": "5.623734327743302",
-              "dailyDistributedTokens": "1000.0000512",
+              "apyPercentage": "4.188904668290827",
+              "dailyDistributedTokens": "750.0000384",
```

Comment:
It seems odd that these would get updated, as they are based on the latest block time anyway (so even after your updates, the block time reference should be the same).